portfolio/src/main/webapp/script.js

```diff
@@ -65,12 +67,21 @@ function checkCommentValidity(event) {
     }
 }
 
+function initMap() {
+    map = new google.maps.Map(document.getElementById('map'), {
+        center: {lat: -34.397, lng: 150.644},
```

Comment:
Maybe add a comment here explaining what location this is. So a different reader / you in the future knows